VideoWeb/Testing.Common/Helpers/VideoWebUriFactory.cs

```diff
@@ -6,14 +6,25 @@ public class VideoWebUriFactory
     {
         public VideoWebParticipantsEndpoints ParticipantsEndpoints { get; }
         public VideoWebCallbackEndpoints CallbackEndpoints { get; }
+        public VideoWebConferenceEndpoints ConferenceEndpoints { get; }
 
         public VideoWebUriFactory()
         {
             ParticipantsEndpoints = new VideoWebParticipantsEndpoints();
             CallbackEndpoints = new VideoWebCallbackEndpoints();
+            ConferenceEndpoints = new VideoWebConferenceEndpoints();
         }
     }
 
+    public class VideoWebConferenceEndpoints
+    {
+        private static string ApiRoot => "conferences";
```

Comment:
Looks like Apiroot is redundant as it is not used anywhere.

Reply:
private static string ApiRoot => "conferences";

        public string GetConferencesForVhOfficer = $"{ApiRoot}/vhofficer";

Reply:
Its used on the next 2 lines... (see above).